entrypoint.bash

```diff
@@ -132,6 +132,14 @@ else
       ./nginx.conf.pre
 fi
 
```

Comment:
minor: for consistency either add an extra space between all the other if/fi blocks or remove this one